docs/labs/installer.rst

```diff
@@ -1,55 +1,87 @@
+* You can now interact with your Flocker cluster using the ``docker`` CLI on the nodes, or locally by using the ``uft-flocker-deploy`` tool or the ``uft-flocker-volumes`` tool.
 
-  * Ubuntu/Debian:
+.. _labs-supported-configurations:
 
-    .. prompt:: bash $
+Supported Configurations
+========================
 
-        sudo apt-get install -y python-pip build-essential libssl-dev libffi-dev python-dev
+This Quick Start Installer can be used in the following configurations.
 
-  * RHEL/CentOS/Fedora:
+* **Supported configurations**
 
-    .. prompt:: bash $
+  * Ubuntu 14.04 on AWS with EBS backend
 
-        sudo yum install -y python-pip gcc libffi-devel python-devel openssl-devel
+..  * Ubuntu 14.04 on Rackspace with OpenStack backend
+..  * Ubuntu 14.04 on private OpenStack cloud with OpenStack backend
+..
+.. * **Experimental configurations**
+..
+..  * CoreOS on AWS with EBS backend
+..  * Ubuntu 14.04 on any infrastructure with ZFS backend
 
-Using ``pip``, you can install ``unofficial-flocker-tools`` straight from GitHub:
+Other configurations (CentOS, OpenStack, ZFS, etc) are possible via the :ref:`official long-form install docs <installing-flocker>`.
 
-.. prompt:: bash $
+You may also be interested in the long-form documentation if you like to see exactly how things are done, or if you're automating setting up Flocker within your own configuration management system.
+
+.. note::
+
+    If you get an error response from any of the commands in this guide, please `report a bug <https://github.com/clusterhq/unofficial-flocker-tools/issues>`_, pasting the ``install-log.txt`` file you will find in the current directory.
+
+.. _labs-installing-unofficial-flocker-tools:
+
+Installing the Installer
+========================
 
-    sudo pip install git+https://github.com/clusterhq/unofficial-flocker-tools.git
+First we install the installer on your workstation.
+This will work on Linux or OS X machines with Docker installed.
 
-If you prefer to use a ``virtualenv``, just activate one and then run ``pip`` without ``sudo``.
+* If you don't have Docker installed, install it now (`Mac <https://docs.docker.com/mac/started/>`_, `Linux <https://docs.docker.com/linux/started/>`_).
+* Then install the installer, which will pull the Docker image:
 
-This will install the following tools on your machine:
+  .. prompt:: bash $
 
-* ``flocker-sample-files``: put some sample ``cluster.yml`` files in the current directory
-* ``flocker-config``: generate certificates and push them
-* ``flocker-install``: install Flocker OS packages on target nodes
-* ``flocker-plugin-install``: install experimental Docker and the :ref:`Flocker Docker plugin <labs-docker-plugin>` on target nodes
-* ``flocker-tutorial``: print out some instructions on how to test the cluster with ``curl`` commands
-* ``flocker-volumes``: an experimental volumes CLI
+      curl -sSL https://get.flocker.io/ | sh
```

Comment:
I added 45c31a67549ba860c94d24275fa3af1a01eba595 to ensure that Docker is running before running the CHQ script.